virttest/qemu_capabilities.py

```diff
@@ -22,6 +22,8 @@ class Flags(object):
     SMP_DIES = _auto_value()
     INCOMING_DEFER = _auto_value()
     MIGRATION_PARAMS = _auto_value()
+    DAEMONIZE = _auto_value()
+    PIDFILE = _auto_value()
```

Comment:
Hi @qingwangrh would you please elaborate on the purpose of adding these two new flags?

Reply:
The QSD is keep evolving ,the options are supported in different version.
I hope QSD may share the capablitlies definition with qemu. So i add them here.

Reply:
The qemu also include relevant options. I can put them in tp-qemu depend on your decision. 

root@qing /home/kar/internal_cfg/guest-os $ /usr/libexec/qemu-kvm  -h|grep daem
-daemonize      daemonize QEMU after initializing
root@qing /home/kar/internal_cfg/guest-os $ /usr/libexec/qemu-kvm  -h|grep pid
-pidfile file   write PID to 'file'

Reply:
> The QSD is keep evolving ,the options are supported in different version. I hope QSD may share the capablitlies definition with qemu. So i add them here.

I see, but I'd suggest making a capablitlies individual definition for the QSD binary due to the correctness concerns and maintainability reasons.

Reply:
Updated